### ISSUE TITLE
Superfluous warnings with Mozilla vendor prefixes in selector-no-vendor-prefix

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#827]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Mozilla (`-moz`) vendor prefixes will show two warnings when detected via the ``selector-no-vendor-prefix` rule, the Microsoft (`-ms`) and Webkit (`-webkit`) only show a single warning.

> Which rule, if any, is this issue related to? 

`selector-no-vendor-prefix`

> What CSS is needed to reproduce this issue?

```css
.webkit-input-placeholder::-webkit-input-placeholder {
	color: red;
}

.moz-placeholder::-moz-placeholder {
	color: red;
}

.ms-input-placeholder:-ms-input-placeholder {
	color: red;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "selector-no-vendor-prefix": true,
  }
}
```

> Which version of stylelint are you using?

 `4.4.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint --config node_modules/stylelint-config-wordpress/index.js src/test.css`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)? 

No

> What did you expect to happen?

Only a single warning flagged for `-moz-placeholder`, not both these warnings `::-moz-placeholder` and `:-moz-placeholder`

```shell
110:17	⚠  Unexpected vendor-prefixed selector "::-webkit-input-placeholder" (selector-no-vendor-prefix) [stylelint]
114:17	⚠  Unexpected vendor-prefixed selector "::-moz-placeholder" (selector-no-vendor-prefix) [stylelint]
119:17	⚠  Unexpected vendor-prefixed selector ":-ms-input-placeholder" (selector-no-vendor-prefix) [stylelint]
```

The following warnings were flagged:

```shell
110:17	⚠  Unexpected vendor-prefixed selector "::-webkit-input-placeholder" (selector-no-vendor-prefix) [stylelint]
114:17	⚠  Unexpected vendor-prefixed selector "::-moz-placeholder" (selector-no-vendor-prefix) [stylelint]
114:18	⚠  Unexpected vendor-prefixed selector ":-moz-placeholder" (selector-no-vendor-prefix) [stylelint]
119:17	⚠  Unexpected vendor-prefixed selector ":-ms-input-placeholder" (selector-no-vendor-prefix) [stylelint]
```

**Comment by davidtheclark** [comment]
Weird. BTW: did you find the prompts helpful or annoying?

**Comment by jeddy3** [comment]
> did you find the prompts helpful or annoying?

Good question! It's nice to see the Issue template being used, but would be great to know if it was helpful to you or a hinderance.  From our perspective, it's a _great_ looking bug report... so thanks for answering all the questions :)

**Comment by ntwb** [comment]
> BTW: did you find the prompts helpful or annoying?

Both annoying and helpful, annoying as it appears as a _wall of text_ within the editor window, helpful in that it reminded me to check via the CLI, as technically I was doing this via a Grunt task, so confirming the issue was reproducible via the CLI and in no way related to other modules in my workflow like `grunt-postcss`, `postcss-reporter, and `postcss-scss` was made it helpful.

> From our perspective, it's a great looking bug report... so thanks for answering all the questions :)

Yeah, I pretty much know what a good bug report should and shouldn't be, but chowing down on ones own dogfood :dog: is important. 

Probably the worst part was deleting all the `E.g.`'s after swapping the example code to my code. :smirk:

For the most part I'd say the issues raised via the original PR [here](https://github.com/stylelint/stylelint/pull/764#issuecomment-185646220) and [here](https://github.com/stylelint/stylelint/pull/764#issuecomment-185662333) are mostly true.

I came upon [this template](https://github.com/Automattic/jetpack/issues/new) in another repo yesterday that makes use of some markdown comments which can only be seen when creating the issue, not sure how we'd use them as nothing explicit comes to mind, just something different I thought I'd point out. Has anyone else seen any awesome GitHub issue templates or ideas to improve ours?

**Comment by ntwb** [comment]
I found an awesome site to generate templates https://www.talater.com/open-source-templates/, the resulting template is OK ([raw](https://gist.githubusercontent.com/ntwb/24bcf0ee639263b45ea0/raw/146d1016c90dd139b1afb55ae11d970fcc3e0b69/temp.md) / [rendered](https://gist.github.com/ntwb/24bcf0ee639263b45ea0) ), but it was all about the journey getting there for me, I smiled the whole way as I was taken back in time to childhood :smiley:

**Comment by ntwb** [comment]
Here's a list of some more examples: 
• https://github.com/devspace/awesome-github-templates#bomb-templates-for-issues

Only two of the list above stood out and were really different to ours or the other examples included there, this [one](https://github.com/bchavez/RethinkDb.Driver/issues/new) uses lists, this [one](https://github.com/driftyco/ionic/issues/new) is quite _clean_ in that after adding your info and code there is not much text/code you have to delete from the original template

**Comment by jeddy3** [comment]
@ntwb Thanks for the all the feed back!

> I found an awesome site to generate templates, the resulting template is OK (raw / rendered ), but it was all about the journey getting there for me,

That site is so good! Thanks for sharing it :)

I've looked through all the templates. We also had a couple of new issues in the last day and the current format of question and answer seems to be working quite well. I'm not sure if that's because of the questions, the example answers or a combination of the two. I'm inclined to keep it as is (despite the chore of deleting all the "e.g"s) for a while longer to see how we get on.

**Comment by ntwb** [comment]
>  I'm inclined to keep it as is (despite the chore of deleting all the "e.g"s) for a while longer to see how we get on.

Agreed, I'm sure we will get more feedback in time
